src/tests/integration/test-data/foundry-prove-skip-legacy

```diff
@@ -103,6 +103,7 @@ ExpectRevertTest.testFail_ExpectRevert_failAndSuccess()
 ExpectRevertTest.testFail_expectRevert_false()
 ExpectRevertTest.testFail_expectRevert_multipleReverts()
 ExternalLibTest.testSquare(uint256)
+ExternalLibTest.testSum(uint256,uint256)
```

Comment:
```suggestion
ExternalLibTest.testSum()
```

Reply:
Fixed in https://github.com/runtimeverification/kontrol/pull/916/commits/490164d73a693eac72fb88a36f7e31a25881329c.